gallery/tutorials/aspire_introduction.py

```diff
@@ -660,7 +662,7 @@ def noise_function(x, y):
 # +----------------+--------------------+-----------------+----------------+---------------------+
 # |CoordinateSource| whiten             |                 | CLSymmetryCn   |                     |
 # +----------------+--------------------+-----------------+----------------+---------------------+
-# |                | phase_flip         |                 |                |                     |
+# |                | phase_flip         |                 | CommonlineSDP  |                     |
```

Comment:
Can add `CLSymmetryC2`.

Reply:
Thanks! I forgot about that one.  Haven't tried it yet :).

Added